plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/PinotColumnHandle.java

```diff
@@ -29,14 +29,22 @@
 {
     private final String columnName;
     private final Type dataType;
+    private final boolean preservePinotEmptyAggregateValue;
```

Comment:
Yep will do: it's because pinot will return 0 for count(*) and distinctcount(*) on an empty set but the other aggregation functions will return -infinity (or default null value) when trino would return null, ex. for sum, min and max.